custom_cards/custom_card_esh_room/README.md

```diff
@@ -53,6 +54,22 @@ This is an alternative room card to the standard one that is more rectangular th
     label: '[[[ return states["sensor.room_temperature"].state + "°C" ]]]'
 ```
 
+## Customizations
+
+To add both temperature and humidity (as shown below) to the card, replace the label section with this code:
+![image](https://user-images.githubusercontent.com/51805396/170957237-4025b2c0-2532-49e6-a908-d45c1f7a0728.png)
+
+```
+'[[[ return states["sensor.room_temperature"].state +"°C " + states["sensor.room_humidity"].state + "%"]]]'
+```
+
+For further customization Emoji can be added to distiguish between multiple sensors:
+
+```
+label: '[[[ return["🌡️"] + states["sensor.room_temperature"].state +"°C  💧"+states["sensor.room_humidity"].state + "%"]]]'
```

Comment:
```yaml
label: >
  [[[
    return "🌡️ " + states[sensor.temperature].state + " °C" + " 💧 " + states[sensor.humidity].state + " %"
  ]]]
```
Is a bit cleaner and hopefully better to understand. You could also mention that they can add more sensors to the same line or interchange the types of sensors by changing the blocks between the plusses `+`

Reply:
@basbruss thanks so much!